pkg/types/azure/machinepool.go

```diff
@@ -0,0 +1,27 @@
+package azure
+
+// MachinePool stores the configuration for a machine pool installed
+// on Azure.
+type MachinePool struct {
+	// Zones is list of availability zones that can be used.
```

Comment:
does Azure have the concept of `Zones` ? coz it would be better to start with the smallest set.

Reply:
Yes it does, only some regions support it, we can add availibility zones support later on.

Reply:
Do you have link to documentation...